### DISCUSSION TITLE
provider_references in In-Network-Rates-Multiple-Plan file

### DISCUSSION THREAD

**Discussion by MRAPAHP** [CMSgov/price-transparency-guide#476] [General]
In the below In-Network-Rates-Multiple-Plan JSON file data, can you please advise the correct way to represent the provider_references to include the provider_group_id and location, so as to pass the Schema validation. We will be using using an external file for the Provider Groups(Provider Reference).

{"reporting_entity_name":"CMS","reporting_entity_type":"CMS","last_updated_on":"2022-05-12","version":"1.0.0","in_network":[{"negotiation_arrangement":"ffs","name":"ABC AGE >17 W CC","billing_code_type":"MS-DRG","billing_code_type_version":"2020","billing_code":"001","description":"ABC AGE >17 W CC","negotiated_rates":[{"provider_references":[1],"negotiated_prices":[{"negotiated_type":"negotiated","negotiated_rate":624.1403,"expiration_date":"2078-12-31T00:00:00","billing_class":"institutional"},{"negotiated_type":"negotiated","negotiated_rate":320.5271,"expiration_date":"2078-12-31T00:00:00","billing_class":"institutional"}]}]}]}

**Comment by olderthandirt5** [comment]
The attached sample shows how we are doing in-network-rates with external provider-reference files. It has passed validation.
[Converted to a .txt extension to allow for upload.]
`in-network-rates_SAMPLE.json v1.0.0 -t in-network-rates

Input JSON is valid.
[in-network-rates_SAMPLE.txt](https://github.com/CMSgov/price-transparency-guide/files/8680991/in-network-rates_SAMPLE.txt)
`

Hope that helps/

**Reply by MRAPAHP** [reply]
Thank you very much. This helps.
